ferrocene/ci/scripts/detect-conflict-markers.py

```diff
@@ -36,6 +36,9 @@
     "<<<<<<< HEAD //~ ERROR encountered diff marker",
     # Introduced in https://github.com/rust-lang/rust/pull/116712.
     ">>>>>>> 7a4f13c blah blah blah",
+    # Example inside rust-dev-guide subtree
+    "<<<<<<< HEAD\n Original code",
```

Comment:
This line doesn't have any effect, since the script checks one line at a time. The line being matched will thus never contain a `\n`.

Including only the last line is enough: only one between the start, middle or end marker needs to be ignored for the whole conflict block to be ignored by the script.